pkg/JuliaInterface/gap/utils.gi

```diff
@@ -4,6 +4,16 @@
 ##
 #############################################################################
 
+BindGlobal( "JuliaSetVal",
+function( name, val )
+  if not IsString( name ) then
+    Error( "JuliaSetVal: <name> must be a string (not the value '", name, "')" );
+  else
+    JuliaFunction( "_setglobal!", "GAP" )( Julia.Main,
```

Comment:
Slightly more efficient:
```suggestion
    Julia.GAP.("_setglobal!")( Julia.Main,
```
Even better: rename that function to e.g. `_setglobal` (without the "!"), then we can write
```suggestion
    Julia.GAP._setglobal( Julia.Main,
```
I guess at this point one could also put this call into a single line :-)